django/db/backends/mysql/features.py

```diff
@@ -72,3 +72,13 @@ def supports_transactions(self):
         All storage engines except MyISAM support transactions.
         """
         return self._mysql_storage_engine != 'MyISAM'
+
+    @cached_property
+    def ignores_quoted_identifier_case(self):
+        """
+        See https://dev.mysql.com/doc/refman/5.7/en/identifier-case-sensitivity.html
```

Comment:
Omit 5.7 in the URL, although this URL is the first result for searching "LOWER_CASE_TABLE_NAMES MySQL" so maybe there's not much value?